langchain4j-chroma/src/test/java/dev/langchain4j/store/embedding/chroma/ChromaEmbeddingStoreIT.java

```diff
@@ -32,7 +32,7 @@
 class ChromaEmbeddingStoreIT extends EmbeddingStoreWithFilteringIT {
 
     @Container
-    private static final ChromaDBContainer chroma = new ChromaDBContainer("chromadb/chroma:0.5.4");
+    private static final ChromaDBContainer chroma = new ChromaDBContainer("chromadb/chroma:latest");
```

Comment:
This is actually less deterministic than targeting the specific version

Reply:
Just in case, this PR is not complete, it is just a reminder for myself to explore this idea.

> This is actually less deterministic than targeting the specific version


But that's the whole point. I think ideally we should:
- test multiple specific versions (and document which versions are tested and supported)
- test the latest version (to have a faster feedback loop when something is breaking in the new version)